ts/src/bigone.ts

```diff
@@ -1709,7 +1829,7 @@ export default class bigone extends Exchange {
         };
     }
 
-    async fetchDeposits (code: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Transaction[]> {
+    async fetchDeposits (code: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<any> {
```

Comment:
@samgermain why are you removing some types from the signature, like Str and the return type?

Reply:
@samgermain only ⬆️  comment is preventing the merge